INSTALL.md

```diff
@@ -56,3 +56,19 @@ dokku$ sudo mkdir -p /var/log/journal
 dokku$ dokku enter job-server
 container$ python manage.py sendtestemail me@myemail.org
 ```
+
+## Rotating the Django secret key
+
+If this is a routine rotation of the secret key,
+as opposed to a compromised secret key,
+you can temporarily leave the secret key to be replaced active:
+
+```bash
+# Store the existing SECRET_KEY as OLD_SECRET_KEY.
+dokku config:get job-server SECRET_KEY
+dokku config:set job-server OLD_SECRET_KEY='xxx'
+# Update the SECRET_KEY to a new, secure value.
+dokku config:set job-server SECRET_KEY='xxx'
+# When sufficient time has elapsed, remove the previous key.
```

Comment:
This troubles me. How much time is sufficient time? How would we know when it had elapsed? I think we need to pick a period in answer to the first question; what does the session expiry policy say? Could we set a reminder in Slack in answer to the second question?

Reply:
I wasn't sure of this either, which is why I originally didn't state anything.

The answer is actually given by the setting for [`SESSION_COOKIE_AGE`](https://docs.djangoproject.com/en/5.0/ref/settings/#session-cookie-age) — which we've not overridden, so it's the default of two weeks.

In case I missed anything, I also confirmed this by locally logging out, logging back in, and checking the most recent `expiry_date` in the `django_session` database table.

Reply:
FWIW, I think it's absolutely acceptable to roll this over sooner, if you prefer. We don't want to log people out if they're in the middle of doing something, but I think it's perfectly reasonable to log out anyone who hasn't interacted with the site in, say, 24 hours. On the other hand, there's not a lot a downside to keeping the fallback key around a bit longer if you'd prefer to do that.